components/script/dom/servointernals.rs

```diff
@@ -52,11 +52,9 @@ impl ServoInternalsMethods<crate::DomTypeHolder> for ServoInternals {
     }
 }
 
-impl RoutedPromiseListener for ServoInternals {
-    type Response = MemoryReportResult;
-
+impl RoutedPromiseListener<MemoryReportResult> for ServoInternals {
     #[cfg_attr(crown, allow(crown::unrooted_must_root))]
```

Comment:
Was crown triggered before due to `Self::`? This might be related to https://github.com/servo/servo/issues/35570 then. Can we remove `#[cfg_attr(crown, allow(crown::unrooted_must_root))]` now that we do not use `Self::` and crown is fixed?